src/components/GameInfoV2/index.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { useEffect, useState } from 'react'
+
+import classNames from 'classnames'
+
+import { DiscordFilled, EditorChoice, Globe, X, Youtube } from '@/assets/images'
+
+import BlockchainsStack from '../BlockchainsStack'
+import Button from '../Button'
+import MetaSection from '../MetaSection'
+import Sticker from '../Sticker'
+import styles from './GameInfoV2.module.scss'
+
+interface EditorChoiceType {
+  isEditorChoice: boolean
+  year?: number
+  className?: string
+}
+export interface GameInfoV2Props {
+  title: string
+  info: {
+    developer?: string
+  }
+  version?: string
+  earlyAccess?: boolean
+  playerCount?: string
+  ImageComponent?: React.ReactNode
+  blockchains?: {
+    chainId: string[]
+    maxVisible?: number
+    showMoreCount?: boolean
+  }
+  socialLinks?: {
+    type: string
+    url: string
+    className?: string
+  }[]
+  editorChoice?: EditorChoiceType
+  actionButton?: React.ReactNode
+  isLoading?: boolean
+  className?: string
+}
+
+const GameInfoV2: React.FC<GameInfoV2Props> = ({
+  title,
+  info,
+  version,
+  earlyAccess,
+  playerCount,
+  ImageComponent,
+  blockchains,
+  socialLinks,
+  editorChoice,
+  actionButton,
+  isLoading,
+  className
+}): JSX.Element => {
+  const [isImageLoading, setIsImageLoading] = useState(isLoading)
+
+  useEffect(() => {
+    if (ImageComponent && !isImageLoading) {
+      setIsImageLoading(false)
+    }
+    if (!ImageComponent && isImageLoading) {
+      setIsImageLoading(true)
+    }
+  }, [ImageComponent, isImageLoading])
+
+  let editorChoiceElement = null
+  if (editorChoice?.isEditorChoice) {
+    editorChoiceElement = (
+      <div className={classNames(styles.editorChoice, editorChoice.className)}>
+        <EditorChoice />
+        Editor&apos;s Choice {editorChoice.year || new Date().getFullYear()}
+      </div>
+    )
+  }
+
+  const socialIcons = (socialLink: string) => {
+    if (socialLink === 'website') {
+      return {
+        icon: Globe,
+        className: styles.websiteIcon
+      }
+    }
+    if (socialLink === 'twitter') {
+      return {
+        icon: X,
+        className: styles.twitterIcon
+      }
+    }
+    if (socialLink === 'discord') {
+      return {
+        icon: DiscordFilled,
+        className: styles.discordIcon
+      }
+    }
+    if (socialLink === 'youtube') {
+      return {
+        icon: Youtube,
+        className: styles.youtubeIcon
+      }
+    }
+  }
+
+  return (
+    <div className={classNames(styles.container, className)}>
+      <div className={styles.imageContainer}>
+        {isLoading ? <div className={styles.imageLoading} /> : ImageComponent}
+      </div>
```

Comment:
I believe we will hide the image on mobile 
![image](https://github.com/user-attachments/assets/fcda22b3-fd7f-45bc-8b4a-bb5abb3cfb64)

I saw your comment on figma but I believe @pimentel08 was referring to the carousel instead of this game image